app/models/patient.rb

```diff
@@ -122,15 +122,50 @@ def latest_transfer
 
   # Patients who are eligible for reminders
   scope :reminder_eligible, lambda {
-    where(purged: false)
-      .where(pause_notifications: false)
+    monitoring_open
+      .joins(:dependents)
       .where('patients.id = patients.responder_id')
-      .where.not('latest_assessment_at >= ?', Time.now.in_time_zone('Eastern Time (US & Canada)').beginning_of_day)
+      .where.not(preferred_contact_method: ['Unknown', 'Opt-out', '', nil])
+      .where(pause_notifications: false)
+      .where(
+        'patients.isolation = ? '\
+        'OR patients.continuous_exposure = ? '\
+        'OR patients.last_date_of_exposure >= ? '\
+        'OR (patients.last_date_of_exposure IS NULL AND patients.created_at >= ?)',
+        true,
+        true,
+        ADMIN_OPTIONS['monitoring_period_days'].days.ago.beginning_of_day,
+        ADMIN_OPTIONS['monitoring_period_days'].days.ago.beginning_of_day
+      )
+      .where(
+        'patients.latest_assessment_at < ? '\
+        'OR patients.latest_assessment_at IS NULL',
+        Time.now.getlocal('-04:00').beginning_of_day
```

Comment:
Shouldn't this be configurable based on the reporting period?

Reply:
I don't know why it was set to the Eastern Time Zone, I don't think it should have been. @holmesie Is there context I'm missing maybe? This will definitely need to change though because `Time.now.getlocal('-04:00')` also assumes Daylight Savings Time. 

I need to think through what this should be. We're checking the timestamp of their latest assessment completion, which is system set in UTC. However, the purpose of the check is to see if they have "filled out an assessment today" - so the question becomes, is that "today" according to UTC or according to the monitoree's timezone? I think if we stick to UTC comparisons and make this "past 24 hours" we should be okay.

Reply:
Was using `Time.now.getlocal('-04:00').beginning_of_day` because it was being used in `report_eligibility()` and `optimal_reminder_eligible` so assumed there was some reason for hard coding the time zone. If the completion time is saved in UTC, then comparing in UTC makes the most sense to me.

Yes, it should be configurable based on the reporting period so that different schedules - like every other day - aren't eligible for reminders every day regardless of the configured period.

Reply:
>Yes, it should be configurable based on the reporting period so that different schedules - like every other day - aren't eligible for reminders every day regardless of the configured period.


There aren't different schedules (like every other day) - it's either a reminder each day or none at all at the moment.

Reply:
+1 how right now there are only daily schedules, but it still feels inconsistent to take the configured period into account in some places (like when determining if symptomatic) but not take it into account here.

Reply:
I don't think I'm following what you mean by "configured period" in this case - the 14 day monitoring period for folks in Exposure?

Reply:
Sorry, I was referencing `reporting_period_minutes` for "How many minutes in a reporting period".